Testscripts/Linux/perf_ntttcp.sh

```diff
@@ -188,6 +181,28 @@ Get_Cyclesperbytes()
 	echo "$cyclesperbytes"
 }
 
+Get_concreatedtime()
```

Comment:
change the name format like other Linux functions in utils.sh.

Reply:
fixed